plugins.d/alarm-notify.sh

```diff
@@ -795,6 +852,17 @@ The source of this alarm is line ${src}"
 
 SENT_PUSHBULLET=$?
 
+# send the twilio SMS
+
+send_twilio "${TWILIO_ACCOUNT_SID}" "${TWILIO_ACCOUNT_TOKEN}" "${TWILIO_NUMBER}" "${to_twilio}" "${host} ${status_message} - ${name//_/ } - ${chart}" "${alarm} 
+Severity: ${severity}
+Chart: ${chart}
+Family: ${family}
+To View Netdata go to: ${goto_url}
+The source of this alarm is line ${src}"
```

Comment:
SMS are normally limited to 140 characters.
Since you are posting a URL and the source line, each notification will need multiple SMS messages. This is a billing issue actually. For each notification sent, twilio will most probably charge you many SMS.

My suggestion is to not send the URL and source via twilio, to keep the message short.